packages/wp-dashboard/src/components/editorSettings/editorSettings.js

```diff
@@ -401,6 +408,11 @@ function EditorSettings() {
                   vendors={vendors}
                   woocommerce={woocommerce}
                 />
+                <PageAdvancement
```

Comment:
Stupid question. Is this a WordPress only feature? Should this code only live in wp-dashboard or should be in the dashboard package?